### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by NickEmma**
## This is how I designed my project
- Added a root element to use re-useable code
- Added a header section for the logo and menu
- Displayed the text in the home page

### CODE REVIEW

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @NickEmma, 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no Pull Request description and is not running the linters checks.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+          "variants",
+          "responsive",
+          "screen"
+        ]
+      }
+    ]
+  },
+  "csstree/validator": true,
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+}
```
Comment (Ginohmk) [inline .stylelintrc.json:21]: - This code will be marked as invalid, please set up your linters correctly by running the commands [here]()

**Review by Ginohmk — COMMENTED** [review]
Hi @NickEmma , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that <EXPLAIN WHY PROJECT IS INVALID>

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .stylelintrc.json (hunk 1 above)
Comment (NickEmma) [reply .stylelintrc.json:21]: Thanks it was helpful and I have resolved the issue and fixed the error

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+:root {
+  --green: #36b37e;
+  --dark-blue: #172b4d;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-transform: capitalize;
```
Comment (Rustamxon7) [inline styles.css:12]: - [ ] Please remove this line because not all of the text should be capitalized 👍🏼

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  color: var(--green);
+}
+
+section .hero p {
+  padding-top: 30px;
+  font-size: 18px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 1.5;
+}
```
Comment (Rustamxon7) [inline styles.css:85]: - [ ] Please change the color of the black part in the h1 text and paragraph in the hero section to `#172B4D` (you can use your --dark-blue variable) 🙌🏼

## Your design
image



## Original design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            <img src="./images/github.png" alt="" />
+          </a>
+          <a href="">
+            <img src="./images/fingers.png" alt="" />
+          </a>
+        </div>
+        <a href="" class="angle">
+          <img src="images/angle.png" alt="" />
+        </a>
+      </div>
```
Comment (Rustamxon7) [inline index.html:45]: - [ ] Please also change `social-icon` `div` to `ul` tag and add `li` elements for each icons 👌🍀

File: README.md [hunk 5]
```diff
@@ -1 +1,50 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project [MIT](./MIT.md) licensed.
```
Comment (Rustamxon7) [inline README.md:50]: - [ ] [OPTIONAL] Let's also add the MIT license file 👋📃

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hi @NickEmma , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (Rustamxon7) [reply README.md:50]: - [ ] Pull request should have a description text. I would recommend adding list of thing that you adding in this repo

File: styles.css (hunk 2 above)
Comment (NickEmma) [reply styles.css:12]: Fix the text transform

File: index.html (hunk 4 above)
Comment (NickEmma) [reply index.html:45]: Made changes to the social icon to use a ul tag

File: README.md (hunk 5 above)
Comment (NickEmma) [reply README.md:50]: okay it is fixed

**Review by danielufeli — APPROVED** [review]
Hi @NickEmma ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Design✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**